src/views/anonymous/Login.js

```diff
@@ -84,7 +84,10 @@ export default function Login() {
                 {
                   networkError && <div className="fr-alert fr-alert--error fr-mt-1w fr-mb-4w">
                     <h3 className="fr-alert__title">&Eacute;chec de la connexion &agrave; Inclusion Connect</h3>
-                    <p className="fr-mb-1v">Merci de v&eacute;rifier votre r&eacute;seau ou contacter le support Inclusion Connect</p>
+                    <p className="fr-mb-1v">
+                      Merci de v&eacute;rifier votre r&eacute;seau ou contacter le support Inclusion Connect par mail au <a className="fr-link fr-link"
```

Comment:
ah mince dans ma suggestion y avait peut etre un doublo fr-link
```suggestion
                      Merci de v&eacute;rifier votre r&eacute;seau ou contacter le support Inclusion Connect par mail au <a className="fr-link"
```

Reply:
juste supprimer la classe en double ;)